scylla/src/frame/response/result.rs

```diff
@@ -475,7 +478,11 @@ fn deser_cql_value(typ: &ColumnType, buf: &mut &[u8]) -> StdResult<CQLValue, Par
                     buf.len()
                 )));
             }
-            CQLValue::Date(buf.read_u32::<BigEndian>()?)
+            let timestamp = buf.read_u32::<BigEndian>()?;
+            //  An unsigned integer representing days with epoch centered at 2^31. (unix epoch January 1st, 1970).
+            let time =
+                NaiveDate::from_ymd(-5877641, 6, 23) + DurationDateTime::days(timestamp.into());
```

Comment:
This magic value is used in two places (here and value.rs) - please put it as a const somewhere (with a descriptive name), along with the comment explaining why it's needed.